planetiler-custommap/pom.xml

```
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>planetiler-custommap</artifactId>
+
+  <parent>
+    <groupId>com.onthegomap.planetiler</groupId>
+    <artifactId>planetiler-parent</artifactId>
+    <version>0.3-SNAPSHOT</version>
```

In the basemap profile, we use 0.4. Without this change, I can't compile the project. But with this change it work...
```suggestion
    <version>0.4-SNAPSHOT</version>
```

---

I wish I'd seen this 5 hours ago 🤕